style/style.css

```diff
@@ -0,0 +1,312 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+body{
+    background-color: #ffffff;
+    display: grid;
```

Comment:
This works fine, but don't forget, HTML 'block' elements like `header`, `div` and `section` will naturally, without any CSS, stack vertically one below the other, and fill the available width. They don't need to be 'held in place' with a grid at `body` level, if that makes sense? eg you could set `min-height: 200px` on the `header` and a bit of `margin-bottom`, and you would achieve the same layout with less CSS. Having said that, it's not really a problem that you've used grid here.